gruntfile.js

```diff
@@ -2,6 +2,7 @@ module.exports = function( grunt ) {
 
 	require( 'matchdep' ).filterDev( 'grunt-*' ).forEach( grunt.loadNpmTasks );
 	require( './.deployment' );
+	var dest = 'dist';
```

Comment:
I added this otherwise Gulp throws an error about `dest` being undefined. I assume in your .deployment file you have stuff that defines where files go, but this wasn't committed and when I added the file, I left it empty.